workers/unity/Assets/Fps/Scripts/UI/Tables/Table.cs

```diff
@@ -0,0 +1,110 @@
+﻿using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Fps
+{
+    public class Table : MonoBehaviour
+    {
+        public RectTransform EntriesParentRect;
+
+        private TableEntry EntryTemplate;
+
+
+        public float EntryHeight
+        {
+            get
+            {
+                if (entryHeightCached >= 0)
+                {
+                    return entryHeightCached;
+                }
+
+                entryHeightCached = EntryTemplate.GetComponent<RectTransform>().rect.height;
+                return entryHeightCached;
+            }
+        }
+
+        private float entryHeightCached = -1f;
+
+        [NonSerialized] private readonly List<TableEntry> Entries = new List<TableEntry>();
```

Comment:
I'm not sure there's any benefit to having [NonSerialized] flag on a private List?

Reply:
probably not